_docs/guides/create-notes-occurrences.md

```diff
@@ -0,0 +1,259 @@
+---
+title: How to create notes and occurrences
+overview: Describes how to create notes and occurrences once you have a running Grafeas server.
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Notes and occurrences are the basic building blocks of storing data in Grafeas.
+Creating and managing them allows you to track information about your project.
+A _[note](/_docs/concepts/what-is-grafeas/overview.md)_ describes high-level
+information, whereas an _[occurrence](/_docs/concepts/what-is-grafeas/overview.md)_
```

Comment:
perhaps be more specific here:

A note describes the high-level information about a metadata and an occurrence describes the occurrence of the note in the project.